Google.PowerShell/Storage/GcsObject.cs

```diff
@@ -993,9 +1000,30 @@ protected override void ProcessRecord()
                 default:
                     throw UnknownParameterSetException;
             }
+
+            string destinationBucket = DestinationBucket ?? gcsObject.Bucket;
+            string destinationObject = DestinationObjectName ?? gcsObject.Name;
+
+            if (!Force)
+            {
+                try
+                {
+                    ObjectsResource.GetRequest objGetReq =
+                        Service.Objects.Get(destinationBucket, destinationObject);
+                    objGetReq.Execute();
+                    // If destination does not exist, jump to catch statment.
+                    if (!ShouldContinue("Object exists. Overwrite?",
```

Comment:
If this is too long for a single line, line break at the first parameter of `ShouldContinue`, otherwise this looks kinda odd IMHO.